mindmeld/models/query_features.py

```diff
@@ -529,7 +529,7 @@ def _extractor(query, resources):
         for entity in system_entities:
             for i in entity.token_span:
                 for j in start_positions:
-                    if 0 < i - j < len(feat_seq):
+                    if 0 <= i - j < len(feat_seq):
```

Comment:
why this change?

Reply:
probably because of VJ's previous change. we should merge to master here?

Reply:
Yeah, I'll rebase this PR